doc/api/http2.md

```diff
@@ -558,11 +558,65 @@ added: REPLACEME
 Calls [`unref()`][`net.Socket.prototype.unref`] on this `Http2Session`
 instance's underlying [`net.Socket`].
 
+### Class: ServerHttp2Session
+<!-- YAML
+added: v8.4.0
+-->
+
+#### serverhttp2session.altsvc(alt[, origin[, stream]])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `alt` {string} A description of the alternative service configuration as
+  defined by [RFC 7838][].
+* `origin` {string} A URL string specifying the origin.
+* `stream` {number} The numeric identifier of a stream (or `0`). Defaults to
+  `0`.
+
+Submits an `ALTSVC` frame (as defined by [RFC 7838][]) to the connected client.
```

Comment:
Would help if what stream ID 0 (and why it must have an origin) could be explained. Not everyone likes to read RFC’s.